### DISCUSSION TITLE
I've made new iOS app for my Azuracast radio station

### DISCUSSION THREAD

**Discussion by pawelzet** [AzuraCast/AzuraCast#4677] [Show and tell]
Hello,

I've made an iOS app of my radio. Application is made from the scratch by me. Features:

-Button for play/pause the stream changing, depending if stream is playing or not
-Player with Artist, title and album art
-Fetching API data for Artist, Song Title and Album Art form the azuracast station API
-Displaying metadata of nowplaying song on the locked screen (Remote Control Center of iOS device)
-Airplay 2 (Beta)

Here are some screenshots:
![IMG_6476](https://user-images.githubusercontent.com/46452256/137115601-5da0ecae-1539-468c-819a-5e4a7f51df85.PNG)
![IMG_6477](https://user-images.githubusercontent.com/46452256/137153506-33dac1f3-dfe5-429b-aac7-bd0eea7e9118.PNG)

Thank you in advance for your opinions and suggestions.

**Comment by pwrj** [comment]
Nice. Have you published the App to the Apple Store yet? Any plans for an Android version? Was it difficult to make the App?

**Reply by amarmashruwala** [reply]
Hey that's great. 
How did you build this? Any GitHub projects t for it? Thanks

**Reply by pawelzet** [reply]
No, for now it's just an iOS app. It was not too hard. You should start trying with the tutorial on YouTube - how to make a music app in Xcode.

**Reply by amarmashruwala** [reply]
Great is this on the App Store?

**Reply by pawelzet** [reply]
It's still on TestFlight because of the licensing issues. Let me know, if you'd like to test it.
